packages/ng/input/src/lib/clearer/clearer.component.html

```diff
@@ -1,4 +1,4 @@
-<a href="#" role="button" class="actionIcon" (click)="onClick($event)">
+<a href="#" role="button" class="clearer" (click)="onClick($event)">
 	<span aria-hidden="true" class="lucca-icon icon-thinCross"></span>
 	<span class="u-mask">Clear</span>
```

Comment:
Missing translation?

Reply:
Indeed. It's out of scope but I'll open an issue.